### DISCUSSION TITLE
Device doesn't show status nor operate door, but all sensors, switches still work?

### DISCUSSION THREAD

**Discussion by tomt5894** [ratgdo/homekit-ratgdo#237] [General]
SOLVED: Changed Settings to Sec+ because the LEARN BUTTON was Purple!!! Sometimes you should just re-read the directions and documentation.

New device, on HomeKit which sees the opener.
Wired up per instructions, Wall switch works correctly, door obstruction sensors work correctly but the device status page has all the status as unknown???  See attached screen shot. How to troubleshoot?

This is a PURPLE learn button Chamberlain Garage Door Opener. I change from Sec2.0+ to Sec+ in the settings.

<img width="856" alt="Screenshot 2024-09-22 at 4 35 21 PM" src="https://github.com/user-attachments/assets/38e7f09c-25a9-4ac9-8c6a-00c04cf1edf9">

**Comment by dkerr64** [comment]
Thanks for reporting that it now works.

**Comment by tomt5894** [comment]
David,

Thanks for closing, might I suggest adding to the PDF instructions to make
sure you have the correct SEC mode for your situation and maybe another
pointer to the GIT docs?

Thanks for a great product.
TomT…
